proxy/backend/spi/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it necessary to add this module `spi`

Reply:
Yes, it is used to support various types of locking and checking operations.